test/container/Dockerfile-Gentoo

```diff
@@ -1,70 +1,53 @@
-# ARG TAG=musl # to enable musl instead of glibc
-
-ARG TAG=systemd
-
 FROM docker.io/gentoo/portage:latest as portage
 
-# kernel and its dependencies in a separate builder
-FROM docker.io/gentoo/stage3:$TAG as kernel
+FROM docker.io/gentoo/stage3:systemd
```

Comment:
Ideally we would test both openrc and systemd, but not sure if this is feasible. Some tests will require functionality from systemd, but most should work in an openrc container as well. And openrc is still more or less "the default" for Gentoo.

Reply:
> Our CI currently not able to handle the '2 containers from one Dockerfile'.

What about making two copies of the Dockerfile, `Dockerfile-Gentoo-systemd` and `Dockerfile-Gentoo-openrc`?

> Given the situation, would you want systemd or OpenRC for Gentoo ?

If we must choose, then I'd personally pick systemd because this should allow testing all features, whereas not everything is going to be possible with openrc.  Though I suspect someone will eventually yell at me for saying this.